src/hooks/useJumperTracking.ts

```diff
@@ -0,0 +1,122 @@
+'use client';
+import { usePathname } from 'next/navigation';
+
+import {
+  JUMPER_ANALYTICS_EVENT,
+  JUMPER_ANALYTICS_TRANSACTION,
+} from 'src/const/abi/jumperApiUrls';
+import { useFingerprint } from './useFingerprint';
+interface JumperDataTrackEventProps {
+  category: string;
+  action: string;
+  label: string;
+  value?: number;
+  data?: { [key: string]: string | number | boolean };
+  isConnected?: boolean;
+  walletAddress?: string;
+  browserFingerprint?: string;
+  isMobile: boolean;
+  sessionId?: string;
+}
+
+const track = async (data: object, path: string) => {
+  try {
+    const response = await fetch(
+      `${process.env.NEXT_PUBLIC_BACKEND_URL}${path}`,
+      {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify(data),
+      },
+    );
+    if (!response.ok && process.env.MODE !== 'production') {
+      console.error('Tracking failed', response.statusText);
```

Comment:
no tracking of errors on prod?

Reply:
Should we leave the logs for debugging in prod? @oktapodia @admsk2